<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="T14s" failures="3" tests="9" name="testclass.client.sesweb.grouptest.ActionFromUserAdvancedMenu" time="727.359" errors="0" timestamp="2024-03-02T14:28:23 ICT" skipped="6">
  <testcase classname="testclass.client.sesweb.grouptest.ActionFromUserAdvancedMenu" name="verifyTheChallengeResponsePage" time="37.688">
    <skipped/>
  </testcase> <!-- verifyTheChallengeResponsePage -->
  <testcase classname="testclass.client.sesweb.grouptest.ActionFromUserAdvancedMenu" name="verifyCheckPBConnexPage" time="225.053">
    <skipped/>
  </testcase> <!-- verifyCheckPBConnexPage -->
  <testcase classname="testclass.client.sesweb.grouptest.ActionFromUserAdvancedMenu" name="verifyResetFailedPrebootLogins" time="237.025">
    <skipped/>
  </testcase> <!-- verifyResetFailedPrebootLogins -->
  <testcase classname="testclass.client.sesweb.grouptest.ActionFromUserAdvancedMenu" name="verifyCheckPBConnexPage" time="37.692">
    <skipped/>
  </testcase> <!-- verifyCheckPBConnexPage -->
  <testcase classname="testclass.client.sesweb.grouptest.ActionFromUserAdvancedMenu" name="verifyResetFailedPrebootLogins" time="37.580">
    <failure message="ERR| Proxy element for: DefaultElementLocator &#039;By.xpath: //button[text()=&#039;Yes&#039;]&#039; Not displayed. Please recheck the locator" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: ERR| Proxy element for: DefaultElementLocator 'By.xpath: //button[text()='Yes']' Not displayed. Please recheck the locator
at org.testng.Assert.fail(Assert.java:111)
at com.clean.ults.ElementController.waitForElementClickAble(ElementController.java:39)
at com.clean.ults.ElementController.click(ElementController.java:47)
at com.clean.pages.users.grouppage.GroupController.clickOnYesBtnOnConfirmationDialog(GroupController.java:77)
at testclass.client.sesweb.grouptest.ActionFromUserAdvancedMenu.verifyResetFailedPrebootLogins(ActionFromUserAdvancedMenu.java:91)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </failure>
  </testcase> <!-- verifyResetFailedPrebootLogins -->
  <testcase classname="testclass.client.sesweb.grouptest.ActionFromUserAdvancedMenu" name="verifyTheChallengeResponsePage" time="37.970">
    <skipped/>
  </testcase> <!-- verifyTheChallengeResponsePage -->
  <testcase classname="testclass.client.sesweb.grouptest.ActionFromUserAdvancedMenu" name="verifyResetFailedPrebootLogins" time="37.583">
    <skipped/>
  </testcase> <!-- verifyResetFailedPrebootLogins -->
  <testcase classname="testclass.client.sesweb.grouptest.ActionFromUserAdvancedMenu" name="verifyTheChallengeResponsePage" time="37.737">
    <failure message="ERR| Proxy element for: DefaultElementLocator &#039;By.id: breadcrumb_User_ChallengeResponse_4&#039; Not displayed. Please recheck the locator" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: ERR| Proxy element for: DefaultElementLocator 'By.id: breadcrumb_User_ChallengeResponse_4' Not displayed. Please recheck the locator
at org.testng.Assert.fail(Assert.java:111)
at com.clean.ults.ElementController.waitForElementVisibility(ElementController.java:27)
at com.clean.ults.ElementController.verify(ElementController.java:60)
at com.clean.pages.users.challengeresponsepage.ChallengeResponseVerifier.isCheckChallengeResponsePageDisplay(ChallengeResponseVerifier.java:17)
at testclass.client.sesweb.grouptest.ActionFromUserAdvancedMenu.verifyTheChallengeResponsePage(ActionFromUserAdvancedMenu.java:42)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </failure>
  </testcase> <!-- verifyTheChallengeResponsePage -->
  <testcase classname="testclass.client.sesweb.grouptest.ActionFromUserAdvancedMenu" name="verifyCheckPBConnexPage" time="39.031">
    <failure message="ERR| Proxy element for: DefaultElementLocator &#039;By.id: breadcrumb_User_CheckPBConnex_4&#039; Not displayed. Please recheck the locator" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: ERR| Proxy element for: DefaultElementLocator 'By.id: breadcrumb_User_CheckPBConnex_4' Not displayed. Please recheck the locator
at org.testng.Assert.fail(Assert.java:111)
at com.clean.ults.ElementController.waitForElementVisibility(ElementController.java:27)
at com.clean.ults.ElementController.verify(ElementController.java:60)
at com.clean.pages.users.checkpbconnexpage.CheckPBConnexVerifier.isCheckPBConnexPageDisplay(CheckPBConnexVerifier.java:17)
at testclass.client.sesweb.grouptest.ActionFromUserAdvancedMenu.verifyCheckPBConnexPage(ActionFromUserAdvancedMenu.java:70)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </failure>
  </testcase> <!-- verifyCheckPBConnexPage -->
</testsuite> <!-- testclass.client.sesweb.grouptest.ActionFromUserAdvancedMenu -->
